### DISCUSSION TITLE
Add infinite animation to marker indicator or point

### DISCUSSION THREAD

**Discussion by GeorgiyShur** [patrykandpatrick/vico#878] [Questions] [answered]
### Question

I'm trying to find a way to add the infinite animation to "current value" of real-time line chart (the last point):

![Screenshot 2024-09-17 at 21 22 32](https://github.com/user-attachments/assets/1e6df9c6-13cd-4120-9b9a-efb40933114d)

I'm stuck a bit, because the marker is drawn on Compose canvas, so even if I put an animator into the marker itself, the UI still won't update without recomposition. I also don't want ho add any hacks to invoke additional recompositions on the whole chart. 

Maybe there is some easy way to do it that I don't see? Otherwise is it possible to get the pixel coordinates of the last entry to draw the animation in compose over the chart?

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @GeorgiyShur.

> Maybe there is some easy way to do it that I don't see? 

Unfortunately, such marker animations aren't supported at the moment. However, we'll be looking into making this possible in the future.

> Otherwise is it possible to get the pixel coordinates of the last entry to draw the animation in compose over the chart?

This will be answered in #884.

**Reply by GeorgiyShur** [reply]
Thanks a lot, looking forward for this support :)
